src/sentry/api/serializers/models/notification_setting.py

```diff
@@ -42,7 +42,7 @@ def get_attrs(
         team_map = {t.id: t for t in item_list if isinstance(t, (Team, RpcTeam))}
         user_map = {u.id: u for u in item_list if isinstance(u, (User, RpcUser))}
 
-        notifications_settings = NotificationSetting.objects._filter(
+        notifications_settings = notifications_service.filter(
```

Comment:
Why do we need to use the RPC service here? Do we serialize notification settings in region silos?

Reply:
Yes, `TeamNotificationSettingsDetailsEndpoint.get` uses `NotificationSettingsSerializer`.